File: sdkwork-ai-sdk-csharp/Models/KnowledgeBaseListResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class KnowledgeBaseListResponse
    {
        public List<KnowledgeBaseResponse>? Data { get; set; }
        public int? Total { get; set; }
        public bool? HasMore { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/knowledge-base-list-response.ts
import type { KnowledgeBaseResponse } from './knowledge-base-response';

/** 知识库列表响应 */
export interface KnowledgeBaseListResponse {
  /** 知识库列表 */
  data?: KnowledgeBaseResponse[];
  /** 总数 */
  total?: number;
  /** 是否有更多 */
  has_more?: boolean;
}
